code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -40,21 +40,29 @@
 	if(istype(H))
 		var/obj/item/organ/external/affected = H.get_organ(user.zone_sel.selecting)
 		if(!affected)
-			user << "<span class='danger'>\The [H] is missing that limb!</span>"
+			to_chat(user, "<span class='danger'>\The [H] is missing that limb!</span>")
 			return
 		else if(affected.robotic >= ORGAN_ROBOT)
-			user << "<span class='danger'>You cannot inject a robotic limb.</span>"
+			to_chat(user, "<span class='danger'>You cannot inject a robotic limb.</span>")
 			return
 
+		if(!H.stat)
+			if(H != user)
+				if(H.a_intent != I_HELP)
+					to_chat(user, "<span class='notice'>[H] is resisting your attempt to inject them with \the [src].</span>")

```

Comment:
This is still going to go "hey, they're resisting!" as soon as you try and inject them. Possibly untrue (what if they're just not on help intent, and not actively trying to resist?), and possibly spammable on both ends - click someone a bunch and then move away before the 3 seconds are up.

Reply:
It's utilitarian. Not everyone who's hostile is in Harm intent, and this needs to be more general than "Maybe the target is resisting, maybe not, I don't know, but it's gonna take a minute".

Reply:
Utilitarian but arbitrary. Not everyone who's resisting is in non-help intent, and not everyone who's on non-help intent is resisting. Or maybe someone's resisting being injected by one person but not another (e.g. you're a merc, you want to be injected by the antag medicborg but not by the chemist with the deathmix). There's a lot of variables at play here, no pun intended, and this seems like it'll just get in the way more often than not.

Reply:
If you've got a better wording, please share. Otherwise it will remain as it is.

Reply:
Set "x is injecting Y with the hypospray" to be visible to both parties, and then "Y resisted!" to be if the timer check fails - or even better, "you couldn't inject Y, stay close to them!" or something.